_fluo-docs-1-2/development/applications.md

```diff
@@ -20,12 +20,12 @@ For both clients and observers, you will need to include the following in your M
 <dependency>
   <groupId>org.apache.fluo</groupId>
   <artifactId>fluo-api</artifactId>
-  <version>1.2.0</version>
+  <version>{{ page.version }}</version>
```

Comment:
Why do this when the docs are under a `1-2` dir?  Are you thinking that in the future we will copy these docs to create the 1.3 docs?

Reply:
Yes. They will be copied to create the 1.3 docs so not having to update this is a big plus.